subsys/net/l2/ieee802154/ieee802154_frame.c

```diff
@@ -291,6 +291,7 @@ validate_mac_command(struct ieee802154_mpdu *mpdu, uint8_t *buf, uint8_t *length
 	case IEEE802154_CFI_UNKNOWN:
 		return false;
 	case IEEE802154_CFI_ASSOCIATION_REQUEST:
+		ar = 1U;
```

Comment:
commit message has a type:  volontarily instead of volutarily. (or willingly, either way).